frontend/src/metabase/visualizations/lib/renderer_utils.js

```diff
@@ -142,16 +142,20 @@ function getParseOptions({ settings, data }) {
   };
 }
 
+function canDisplayNull(settings) {
+  // histograms are converted to ordinal scales, so we need this ugly logic as a workaround
```

Comment:
https://github.com/metabase/metabase/blob/master/frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js#L817

☝️ That's the culprit. I thought about unwinding it, but that seemed like too big of a change for my penultimate day at Metabase.